superset/translations/pt/LC_MESSAGES/message.po

```diff
@@ -0,0 +1,4181 @@
+msgid "Pandas resample how"
+msgstr "Pandas remisturar como"
+
+#: superset/assets/javascripts/explore/stores/controls.jsx:619
+msgid "Resample Fill Method"
+msgstr "Método de Preenchimento da Remistura"
+
+#: superset/assets/javascripts/explore/stores/controls.jsx:622
+msgid "Pandas resample fill method"
+msgstr "Método de preenchimento da remistura de pandas"
+
+#: superset/assets/javascripts/explore/stores/controls.jsx:628
+msgid "Since"
+msgstr "Desde"
+
+#: superset/assets/javascripts/explore/stores/controls.jsx:629
+msgid "7 days ago"
+msgstr "Há 7 dias"
+
+#: superset/assets/javascripts/explore/stores/controls.jsx:635
+msgid "Until"
+msgstr "Até"
+
+#: superset/assets/javascripts/explore/stores/controls.jsx:642
+msgid "Max Bubble Size"
+msgstr "Tamanho máximo da bolha"
+
+#: superset/assets/javascripts/explore/stores/controls.jsx:650
+msgid "Whisker/outlier options"
+msgstr "Opções de Whisker / Outlier"
+
+#: superset/assets/javascripts/explore/stores/controls.jsx:652
+msgid "Determines how whiskers and outliers are calculated."
+msgstr "Determina como são calculados os whiskers e os outliers."
+
+#: superset/assets/javascripts/explore/stores/controls.jsx:663
+msgid "Ratio"
+msgstr "Rácio"
+
+#: superset/assets/javascripts/explore/stores/controls.jsx:666
+msgid "Target aspect ratio for treemap tiles."
+msgstr "Aspeto do rácio do alvo para blocos do treemap."
+
+#: superset/assets/javascripts/explore/stores/controls.jsx:672
+#: superset/assets/javascripts/explore/stores/visTypes.js:643
+#: superset/assets/javascripts/explore/stores/visTypes.js:668
+#: superset/assets/javascripts/explore/stores/visTypes.js:817
+msgid "Number format"
+msgstr "Formato numérico"
+
+#: superset/assets/javascripts/explore/stores/controls.jsx:682
+msgid "Row limit"
+msgstr "Limite de linha"
+
+#: superset/assets/javascripts/explore/stores/controls.jsx:690
+msgid "Series limit"
+msgstr "Limite de série"
+
+#: superset/assets/javascripts/explore/stores/controls.jsx:693
+msgid "Limits the number of time series that get displayed"
+msgstr "Limita o número de séries temporais que são exibidas"
+
+#: superset/assets/javascripts/explore/stores/controls.jsx:698
+msgid "Sort By"
+msgstr "Ordenar por"
+
+#: superset/assets/javascripts/explore/stores/controls.jsx:700
+msgid "Metric used to define the top series"
+msgstr "Métrica usada para definir a série superior"
+
+#: superset/assets/javascripts/explore/stores/controls.jsx:708
+msgid "Sort Descending"
+msgstr "Ordenar decrescente"
+
+#: superset/assets/javascripts/explore/stores/controls.jsx:710
+msgid "Whether to sort descending or ascending"
+msgstr "Ordenar de forma descendente ou ascendente"
+
+#: superset/assets/javascripts/explore/stores/controls.jsx:715
+#, fuzzy
+msgid "Rolling"
+msgstr "Rolling"
+
+#: superset/assets/javascripts/explore/stores/controls.jsx:718
+msgid ""
+"Defines a rolling window function to apply, works along with the [Periods] "
+"text box"
+msgstr ""
+
+#:
```

Comment:
There's an extra space after "seleção", should be "Selecione para incluir seleção de granularidade do Druid".

Reply:
agreed